uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerMicrosoftAuthenticator.java

```diff
@@ -63,9 +64,12 @@ public class BrokerMicrosoftAuthenticator extends AbstractTestBroker implements
 
     private final static String INCIDENT_MSG = "Broker Automation Incident";
 
-    public static final String TAG = BrokerMicrosoftAuthenticator.class.getSimpleName();
+    private static final String TAG = BrokerMicrosoftAuthenticator.class.getSimpleName();
+
+    protected boolean isInSharedDeviceMode = false;
+
+    public static BrokerMicrosoftAuthenticator brokerMicrosoftAuthenticatorImpl;
```

Comment:
I don't think we want this to be a static that is set from a different class

Reply:
This should really a Non-Static, `protected` member variable that is set directly in the `install` method. It doesn't need to be set from anywhere else

Reply:
I am now using a private member and setting it directly from BrokerMicrosoftAuthenticator only.